src/release/breaking-changes/ignoringsemantics-migration.md

```diff
@@ -13,7 +13,12 @@ The `ignoringSemantics` properties are removed in [`IgnorePointer`][],
 
 The `ignoreSemantics` was introduced as a workaround to mitigate the result of
```

Comment:
I am not completely grokking this PR. Maybe we should have a quick video meeting? It needs more explanation.